clients/java/src/main/java/io/camunda/client/CamundaClientBuilder.java

```diff
@@ -145,6 +145,12 @@ CamundaClientBuilder jobWorkerExecutor(
   /** The request timeout used if not overridden by the command. Default is 10 seconds. */
   CamundaClientBuilder defaultRequestTimeout(Duration requestTimeout);
 
+  /**
+   * The {@link io.camunda.client.impl.command.ActivateJobsCommandImpl}'s response timeout offset.
+   * Default is 1 second.
```

Comment:
🔧 Let's provide a bit more context what this does, e.g. why this offset is needed.

Reply:
Hm, hard to say, actually. We set the response timeout in the HTTP request, not the request timeout. For the client user it might be confusing though to use response timeout here now.

Reply:
You have a good explanation in `ActivateJobsCommandStep1`. Let's move this here maybe, and reference it from there?
I was also wondering if we should name this more generally, maybe just `defaultRequestTimeoutOffset`? We can explain that this can be used by commands if they pass on the timeout to the called server to avoid the client request timing out before it receives a response due to network latency. We can state that this is not used by all commands by default but just in such cases. Wdyt?

Reply:
We could make it generic, maybe something else will use this, too. I'll also add the list the commands that use this, right now only ActivateJobs.

Reply:
I would vote for not listing them in the docs. It will get outdated and you can also search for code usages and find the command(s) that use it, I guess?

Reply:
They could for sure find it out from the code, I just wanted to be nice. 😄

Reply:
Yeah, I know, but it will backfire, I can already feel it 🤣

Reply:
Dropped it